Chinese/16fall_USA/Columbia_university/columbia I am coming.md

```diff
@@ -0,0 +1,4 @@
+写于2016/4/13 比利时布鲁塞尔自由大学图书馆
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.